blocks/library/latest-posts/index.php

```diff
@@ -52,7 +52,7 @@ function gutenberg_render_block_core_latest_posts( $attributes ) {
 		$class .= ' is-grid';
 	}
 
-	if ( isset( $attributes['columns'] ) ) {
+	if ( isset( $attributes['columns'] ) && 'grid' === $attributes['layout'] ) {
```

Comment:
Based on the surrounding code, it looks like this should check to make sure the `layout` key exists before checking its value.

Reply:
I think we can remove all isset() from this file, because the attributes were moved to the server in #2529
cc: @aduth

Reply:
We might still want the `'grid' === $attributes['layout']` check though since based on the behavior in the editor, it seems that columns are only relevant for the grid layout.